src/components/config/DynamicForm.vue

```diff
@@ -95,10 +95,9 @@
             })
           : [];
       },
-    },
-    setSelectedInput(index, value) {
-      this.selectedInputs[index] = value;
-      this.emitInput();
+      handleUpload(value, index) {
+        (this.selectedInputs[index] = value), this.emitInput();
```

Comment:
For legibility, better to let each statement in a unique line no?

Reply:
It is in a unique line on original code

Reply:
What does "original code" mean?